engine/remote-components/src/main/java/org/datacleaner/components/remote/RemoteTransformer.java

```diff
@@ -120,9 +125,13 @@ public OutputColumns getOutputColumns() {
                         try {
                             outCols.setColumnType(i, Class.forName(colSpec.type));
                         } catch (ClassNotFoundException e) {
-                            // TODO: what to do with data types - classes that are not on our classpath?
-                            // Provide it as pure JsonNode?
-                            outCols.setColumnType(i, JsonNode.class);
+                            Class type;
+                            if (isOutputColumnEnumeration(colSpec.schema)) {
+                                type = EnumerationValue.class;
```

Comment:
Shouldn't we rather return the enum values as strings? After all the client side cannot use any kind of type-based mapping of columns when the EnumerationValue class is used (because it is non-concrete). At least he can do string based mapping of columns instead.